tests/test_helpers.py

```diff
@@ -135,3 +135,19 @@ async def wait_for_event(websocket, event_method):
         if "method" in event_response and event_response[
                 "method"] == event_method:
             return event_response
+
+
+def AnyExtending(expected):
```

Comment:
Would be nice to document this.

Also, this seems to re-create `recursive_compare`. Did you check if the diffs lock nicer when the test fails? Otherwise we may just have walked in a circle

Reply:
https://github.com/GoogleChromeLabs/chromium-bidi/pull/560